apps/src/applab/commands.js

```diff
@@ -19,6 +19,7 @@ import {
 } from '../lib/util/javascriptMode';
 import {commands as audioCommands} from '@cdo/apps/lib/util/audioApi';
 import {commands as timeoutCommands} from '@cdo/apps/lib/util/timeoutApi';
+import {commands as mlCommands} from '@cdo/apps/lib/util/mlApi';
```

Comment:
I see a lot of "ml" related naming, but it sounds like we are going to trend towards "ai" instead?

Reply:
Yeah - it'd be good to get all on the same page about that and update the code to match.  Ok to do as a rename/refactor after this PR?

Reply:
Sounds fine.